InstanceExport/PowerShell/README.md

```diff
@@ -27,8 +27,63 @@ The process will prompt to you the mandatory fields to be typed in your screen `
+
+```
+An error occurred
+Error occurred when extracting data from /api/awards/123/documents/Evaluation/NewProject/Baseline/Folder Name with ’ invalid character/My file for reports on May 2018.docx
+StatusCode: 500
+Url: /api/awards/123/documents/Evaluation/NewProject/Baseline/Folder Name with ’ invalid character/My file for reports on May 2018.docx
+StatusDescription: Internal Server Error
+```
+
+- This error message means that the Instance Export script was not able to download the file. It is most probably due to the ’ character or any other invalid character that exist currently in the folder name or else in the file name. Our recommendation to solve the problem is: please try to change the name of the file and remove any special characters like for example ’, ¿, ™ and try to run the script again and see if the error goes away
+
+```
+An error occurred
+Error occurred when extracting data from /api/awards/123/documents/Annual Reviews 2020/FileName with a ’ character that may be ´ invalidçã£.pptx
+StatusCode: 404
+Url: /api/awards/123/documents/Annual Reviews 2020/FileName with a ’ character that may be ´ invalidçã£.pptx
+StatusDescription: Not Found
+```
+
+- This error message means that the Instance Export script was not able to find the data/file to be exported. It can indicate two things: 1. The award probably could have been deleted 2. We couldn't find the document because it can contain a invalid character in the filename. Our recommendation to solve the problem is: please try to check if the name of the file has any special characters like for example ’, ´ , £ and remove it from the name. Then try to run the script again and see if the error goes away. If the error persists, please contact us through help desk and we can investigate it further and help you with it
+
+```
+An error occurred
+Error occurred when extracting data from /api/awards/123/documents/O1. Project Development/O8.1 Evidence/Extra CP Messages 10.04.2055¿Edited-Approved Messages.docx
+StatusCode: 400
+Url: /api/awards/123/documents/O1. Project Development/O8.1 Evidence/Extra CP Messages 10.04.2055¿Edited-Approved Messages.docx
+StatusDescription: Bad Request
+```
+
+- This error message means that the Instance Export script sent a request to our API and it was not processed because we couldn't find the document because it contains a invalid character like a ¿ or something else in the filename that was not accept as a valid request in our API server. Our recommendation to solve the problem is: pleas contact us through help desk and we can investigate it furter and help you to solve the problem but it might be related to renaming the file and removing the invalid characters in the file name.
```

Comment:
```suggestion
- This error message means that our server could not find the document. This could happen because the document file name contains an invalid character like a `¿` or something else that was not accepted as a valid request by our server. Our recommendation to solve the problem is: please contact us through help desk and we can investigate it further and help you to solve the problem but it might be related to renaming the file and removing the invalid characters in the file name.
```